webapp/src/components/ActivityPage/Transaction/Transaction.js

```diff
@@ -351,7 +351,7 @@ export default class Transaction extends React.PureComponent {
   }
 
   renderEstatePreview(tx) {
-    const size = 5
+    const size = 9.6
```

Comment:
Same about the constant

Reply:
This is just because the size of the preview is 48 pixels (48x48), and we show 5x5 parcels in it, so 48 / 5 = 9.6

I can refactor a bit to make it more clear tho